index.html

```diff
@@ -667,7 +667,7 @@ <h2 class="accent-text">Nov 17th, 2023 @ DevConnect Istanbul</h2>
     <div class="page-content">
         <section class="conference-pitch">
             <div class="intro-text">
-                <p>WalletUncon is a one day unconference on blockchain wallets, standards, usability, and more. Come learn from some foundational overview  talks, and then split into groups for discussion.</p>
+                <p>WalletUncon is a one day unconference on wallets, standards, usability, and more. Come learn from some foundational overview  talks, and then split into groups for discussion.</p>
                 <p>We'll be hosted inside of the DevConnect space, and will be the day after the Web3 UX Unconference.</p>
```

Comment:
use the canonical way of writing it - I also like CamelCase and DEVConnect more - but the canonical way is Devconnect
also  "Web3 UX Unconference" is not yet publicly announced - so removing it

```suggestion
                <p>We'll be hosted inside of the Devconnect space.</p>
```